domain/src/main/java/com/poetry/impl/PoetryReader.java

```diff
@@ -1,9 +1,20 @@
 package com.poetry.impl;
 
+import com.poetry.port.ObtainPoem;
 import com.poetry.port.RequestVerse;
 
 public class PoetryReader implements RequestVerse {
+    ObtainPoem obtainPoem;
+    public PoetryReader() {
+        this.obtainPoem = new ObtainPoem() {
+        };
+    }
+    public PoetryReader(ObtainPoem obtainPoem) {
+        this.obtainPoem = obtainPoem;
+    }
+
+    @Override
     public String getVerse() {
-        return "Hello Nirmala";
+        return obtainPoem.getVerseFromPoem();
```

Comment:
Need to add some business logic in the service.
```suggestion
        return obtainPoem.getVerseFromPoem() + "\n";
```